decidim-proposals/lib/decidim/proposals/test/factories.rb

```diff
@@ -124,6 +124,14 @@
       answer { Decidim::Faker::Localized.sentence }
       answered_at { Time.current }
     end
+
+    trait :draft do
+      published_at nil
+    end
+
+    trait :published do
+      published_at { Time.current }
```

Comment:
I'd have set the `published_at` by default instead of adding `published` every time.

Reply:
yeah :dizzy_face: maybe it would have been more practical